src/core/css/inc/mixins.scss

```diff
@@ -58,6 +58,18 @@
   outline: none;
 }
 
+@mixin disabled($background) {
+  @if lightness($background) > 80 {
+    background: adjust-hue(darken(desaturate($background, 85%), 13%), 6);
+  }
+
```

Comment:
That's exactly how it was before I moved it into a function. It triggers a lint error now that it's in a function. I see other instances inside functions that do not trigger an error. Shrug. It must be a bug in stylelint.